### ISSUE TITLE
Default Currency is wrong

### ISSUE THREAD

**Issue by MisterY** [sdementen/piecash#33]
I may need to investigate further, but I'm getting an unexpected result.

```
with piecash.open_book(file_path, open_if_lock=True) as book:
    default_currency = book.default_currency
    print("Default currency is ", default_currency.mnemonic)
```
This gives me GBP. When I open GnuCash and go to Preferences/Accounts, under Default Currency there are two options:
- Locale: GPB (Pound Sterling)
- Choose: EUR (Euro)

My selection is on the second option, which means that the default should be Euro. However, the book.default_currency is giving back the Locale one.

**Comment by MisterY** [comment]
Aha, I see what's going on [here](https://github.com/sdementen/piecash/blob/76cc6e6c9b74ec28038c7792650e1bff0b045f18/piecash/core/book.py#L135).
This is probably working for most but not in my case, for some reason. There must be some storage where my selected currency is kept, though.

**Comment by sdementen** [comment]
yes, I have no clue where Gnucash is storing (or guessing) this default currency. I have looked at many places but no luck. If you find gnucash rules, the `default_currency` could be updated accordingly

**Comment by MisterY** [comment]
Will do. First [trace](https://github.com/Gnucash/gnucash/blob/723530a9bc212a618bf6adf69d88fb9f8789ca17/gnucash/gnome/gschemas/org.gnucash.gschema.xml.in.in#L128).

**Comment by MisterY** [comment]
Got it. See above. :D

**Comment by sdementen** [comment]
good! so my implementation was for the "default" case when the key is not defined.
now, also to get the logic on MacOSX and Linux ...

**Comment by MisterY** [comment]
[This](https://stackoverflow.com/questions/24407025/how-do-i-use-g-settings-schema-get-key-from-python) might help.
I guess there is a working wrapper for GSettings.

**Comment by sdementen** [comment]
Can you have a try on a PR for this ? 
my next working item will be the reporting logic

**Comment by MisterY** [comment]
There are some complications with installing Gtk and Gio for Windows & Python 3. :(
Will look into this further.

**Comment by sdementen** [comment]
for windows, I guess gnucash use the windows registry (https://docs.python.org/3.1/library/winreg.html) so that will be the easiest :-)

**Comment by MisterY** [comment]
[This](https://github.com/MisterY/gnucash-portfolio/blob/master/examples/windows_registry.py) gives:
```
currency-choice-locale [0, 4]
currency-choice-other [1, 4]
currency-other ['EUR', 1]
```
in my case. Yay!

**Comment by MisterY** [comment]
Submitted #34.

**Comment by sdementen** [comment]
According to @jralls email of today, the default currency of a book is the currency of the ROOT account.
The default currency when creating a new book is related to the localeand the registry. However, creating a book in gnucash should always ask for the default currency of the new book (i.e. the root account currency)

**Comment by sdementen** [comment]
Fixed in 0.17.0 by returning the ROOT account currency
